[INTERNAL] docs/pages/Builder.md: Explain bundling of files requiring top level scope (#975)
SAP/ui5-tooling: docs/pages/Builder.md

## Legacy Bundle Tooling (lbt)
JavaScript port of the "legacy" Maven/Java based bundle tooling.

### JavaScript files requiring top level scope

> UI5 Tooling packages JavaScript files requiring top level scope as a string as long as your project using specVersion lower than `4.0`. This stringified code is evaluated via [`eval`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval) at runtime. This behavior ensures that the script works as expected, e.g. with regards to implicit globals. However this `eval` runtime feature will be discontinued with UI5 2.x because of security best practices (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval) and to comply with stricter CSP settings (`unsafe-eval`). When your project defines [Specification Version 4.0](./Configuration.md#specification-version-40) or higher files requiring top level scope are no longer part of the created bundle and an error is logged by UI5 Tooling. Therefore, please adjust your code with the help of one of the following options:

Reviewer: ```suggestion
UI5 Tooling packages JavaScript files that require "top level scope" as a string, as long as your project is using a Specification Version lower than `4.0`. In this case, the respective code is evaluated using [`eval`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval) at runtime.

This behavior ensures that the script works as expected, e.g. with regards to implicitly used globals. However this `eval` runtime feature will be discontinued with UI5 2.x because of security best practices (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval) and to comply with stricter CSP settings (`unsafe-eval`). When your project defines [Specification Version 4.0](./Configuration.md#specification-version-40) or higher, files requiring top level scope are no longer part of the created bundle and an error is logged by UI5 Tooling. Therefore, please adjust your code with the help of one of the following options:
```

## Legacy Bundle Tooling (lbt)
JavaScript port of the "legacy" Maven/Java based bundle tooling.

> ### JavaScript files requiring top level scope

Reviewer 2: ```suggestion
### JavaScript Files Requiring Top Level Scope
```

> UI5 Tooling packages JavaScript files that require "top level scope" as a string, as long as your project is using a Specification Version lower than `4.0`. In this case, the respective code is evaluated using [`eval`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval) at runtime.

Reviewer 2: ```suggestion
UI5 Tooling packages JavaScript files that require "top level scope" as a string, provided your project uses a Specification Version lower than `4.0`. In this case, the code is evaluated using [`eval`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval) at runtime.
```

> This behavior ensures that the script works as expected, e.g. with regards to implicitly used globals. However this `eval` runtime feature will be discontinued with UI5 2.x because of security best practices (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval) and to comply with stricter CSP settings (`unsafe-eval`). When your project defines [Specification Version 4.0](./Configuration.md#specification-version-40) or higher, files requiring top level scope are no longer part of the created bundle and an error is logged by UI5 Tooling. Therefore, please adjust your code with the help of one of the following options:

Reviewer 2: ```suggestion
This ensures that the script works as expected, e.g. with regards to implicitly used globals. However, this `eval` runtime feature will be discontinued with UI5 2.x because of security best practices (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval) and to comply with stricter CSP settings (`unsafe-eval`). If your project defines [Specification Version 4.0](./Configuration.md#specification-version-40) or higher, files requiring top level scope are no longer part of the created bundle and an error is logged by UI5 Tooling. Therefore, please adjust your code with one of the following options:
```
